includes/Core/Modules/Module.php

```diff
@@ -518,15 +518,29 @@ private function validate_data_request( Data_Request $data ) {
 			return;
 		}
 
-		$datapoint = $definitions[ $datapoint_key ];
-
-		// If the datapoint requires specific scopes, ensure they are satisfied.
-		if ( ! $this->authentication->get_oauth_client()->has_sufficient_scopes( $datapoint['scopes'] ) ) {
-			$request_scopes_message = ! empty( $datapoint['request_scopes_message'] )
-				? $datapoint['request_scopes_message']
-				: __( 'You’ll need to grant Site Kit permission to do this.', 'google-site-kit' );
+		$datapoint        = $definitions[ $datapoint_key ];
+		$requires_service = isset( $datapoint['service'] );
```

Comment:
`isset` will return `true` even for an empty string, for which most `service` definitions are set to that don't rely on one.
```suggestion
		$requires_service = ! empty( $datapoint['service'] );
```